docs/guides/mobile-plugin-gma-android.md

```diff
@@ -0,0 +1,58 @@
+# UID2 GMA Plugin for Android Integration Guide
+
+## Introduction
+
+The UID2 Google Mobile Ads (GMA) Plugin for Android enables publishers that use the [Google Mobile Ads (GMA) SDK](https://developers.google.com/ad-manager/mobile-ads-sdk) to include UID2 data in ad requests from Android apps. This is done automatically, with no direct coding needed from the app developer.
```

Comment:
"UID2 data" or "UID2 token" or "UID2 identity token"?  i just feel data doesn't sound very specific.

Reply:
Linked to glossary def. Great idea. Thank you! In both files.
"enables publishers that use the [Google Mobile Ads (GMA) SDK](https://developers.google.com/ad-manager/mobile-ads-sdk) to include UID2 data in ad requests from"

Reply:
the link to token definition doesn't seem to work on my Intellij IDE render of the page but if you think it would work then that's fine.